apps/mobile/src/store/accounts/utils.ts

```diff
@@ -14,4 +15,5 @@ export interface AccountStore {
   icon: AvatarIconName;
   name: string;
   status: AccountStatus;
+  type?: WalletStore['type'];
```

Comment:
Adding this as we have some code in `AccountCardLayout` to show `LedgerBadge` if ` type === 'ledger'`. 

We aren't using this now though so I am inclined to remove it until we add ledger support to mobile